azure-kusto-ingest/azure/kusto/ingest/_streaming_ingest_client.py

```diff
@@ -97,12 +97,16 @@ def _ingest(self, stream, size, ingestion_properties, content_encoding=None):
         ):
             raise KustoMissingMappingReferenceError()
 
+        client_request_properties = ClientRequestProperties()
+        client_request_properties.set_option("content_length", str(size))
```

Comment:
If those are relevant for all streaming ingestions, maybe the right place for them is in the request module.